docker/android/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM ubuntu:zesty
+# sys_image can be overridden from the CLI
+ARG sys_image=debian:jessie
```

Comment:
Ok, please let me know if this problem you mentioned then goes away after the docker upgrade 😄 
Thanks